README.md

```diff
@@ -17,6 +17,8 @@ This project relies heavily on the work by [pgvector](https://github.com/pgvecto
 
 ---
 
+[![Tembo Cloud Try Free](https://tembo.io/tryFreeButton.svg)](https://cloud.tembo.io/sign-up)
```

Comment:
can you center it under the main image defined on line 6?

Reply:
Good call, updated.